local.properties

```diff
@@ -1,10 +1,8 @@
-## This file is automatically generated by Android Studio.
-# Do not modify this file -- YOUR CHANGES WILL BE ERASED!
-#
-# This file should *NOT* be checked into Version Control Systems,
+## This file must *NOT* be checked into Version Control Systems,
 # as it contains information specific to your local configuration.
 #
 # Location of the SDK. This is only used by Gradle.
 # For customization when using a Version Control System, please read the
 # header note.
-sdk.dir=/Users/psmattas/Library/Android/sdk
\ No newline at end of file
+#Fri Oct 27 10:59:25 IST 2023
+sdk.dir=/Users/nainsha/Library/Android/sdk
```

Comment:
Don't think this file should be changed as per header comment *Do not check into version control* Potentially we can delete this file from the PR? Open do discuss :)

Reply:
Yeah this file has accidentally been pushed, I'll remove it from the codebase

Reply:
Great thank you